bin/docker-reset

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+# DOC: Stop ALL docker containers and destroy ALL containers, volumes, networks.
+
+source bin/lib.sh
```

Comment:
NB: Do we need to source this for this script?

Reply:
It would probably work fine without it, but even if we don't use the functions defined in `bin/lib.sh` it does set some good default bash options e.g. `set -e` which we generally want to be consistent across all the scripts